apps/fluent-tester/src/TestComponents/Input/InputTest.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+
+import { InputDefault } from './InputDefault';
+import { INPUT_TESTPAGE } from '../../../../E2E/src/Input/consts';
+import type { TestSection, PlatformStatus } from '../Test';
+import { Test } from '../Test';
+
+const inputSections: TestSection[] = [
+  {
+    name: 'Input Page',
+    testID: INPUT_TESTPAGE,
+    component: InputDefault,
+  },
+];
+
+export const InputTest: React.FunctionComponent = () => {
+  const status: PlatformStatus = {
+    win32Status: 'Backlog',
+    uwpStatus: 'Backlog',
+    iosStatus: 'Experimental',
```

Comment:
Dropping a point for review here - if we go ahead with iOS validations for basic structure, this holds true - else we would like to update this back to Backlog.

Reply:
Changing iOS to backlog based on offline discussions.